pages/blog/_slug.vue

```diff
@@ -218,6 +218,7 @@ export default {
         mainImageAlt:
           this.article.coverImgAlt ||
           'Form builder showing drag and drop functionality',
+        keywords: this.article.keywords
       }
       return getSiteMeta(metaData)
     },
```

Comment:
below we have this code is it not working?
if not please remove that code      

 ```
    {
          name: 'keywords',
          content: this.article.keywords,
        },
```

Reply:
Yes, that one is not working since we are calling getSiteMeta in computed property. Fixed